src/NodeChanger/MethodNameChanger.php

```diff
@@ -10,12 +10,29 @@
 {
     /**
      * @param string[] $renameMethodMap
+     * @param string[]|string[] $acceptedAssertionMethods
      */
-    public function renameNode(MethodCall $node, array $renameMethodMap): ?Node
-    {
+    public function renameNode(
+        MethodCall $node,
+        array $renameMethodMap,
+        array $acceptedAssertionMethods = [],
+        ?string $activeFuncCallName = null
+    ): ?Node {
         $oldNodeMethodName = $node->name->toString();
 
-        $node->name = new Identifier($renameMethodMap[$oldNodeMethodName]);
+        if (empty($acceptedAssertionMethods)) {
+            $node->name = new Identifier($renameMethodMap[$oldNodeMethodName]);
+
+            return $node;
+        }
+
+        [$trueMethodName, $falseMethodName] = $renameMethodMap[$activeFuncCallName];
+
+        if (in_array($oldNodeMethodName, $acceptedAssertionMethods['trueMethodNames']) && $trueMethodName) {
```

Comment:
`trueMethodNames` also looks as PHPUnit related, it should be independent.

Reply:
If it is impossible to abstract from PHPUnit, create a new method with name that clearly states this.

Also verify `trueMethodNames` key is present, so we don't get "Missing `trueMethodNames` key" errors.